src/components/BoardInfo.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+    <table>
+        <thead>
+            <tr>
+                <th></th>
+                <th v-for="list in lists" v-bind:key="list.id">{{list.name}}</th>
+            </tr>
+        </thead>
+        <tbody>
+            <tr>
+                <th>Number of Cards</th>
+                <td v-for="list in lists" v-bind:key="list.id + '_count'">{{ cardCountByList[list.id].length }}</td>
+            </tr>
+        </tbody>
+    </table>
+</template>
+
+<script>
+export default {
+  name: 'BoardInfo',
+  data() {
+    return {
+      lists: [],
+      cardCountByList: {},
+      listIds: [
+        '5bcf863f74837934564848c2',
+        '5bcf863f74837934564848c3',
+        '5bcf863f74837934564848c4',
+        '5bcf863f74837934564848c5',
+        '5bcf863f74837934564848c6',
+        '5bcf863f74837934564848c7',
+        '5bcf863f74837934564848c8',
+      ],
+      listIncludesArchived: ['5bcf863f74837934564848c8'],
+    };
+  },
+  props: {
+    boardId: String,
+  },
+  mounted() {
+    this.getLists(this.$props.boardId, this.listIds);
+  },
+  methods: {
+    getLists(boardId, listIds) {
+      const self = this;
+      const url = `boards/${boardId}/lists`;
+      window.Trello.rest('get', url, data => {
+        for (let index = 0; index < data.length; index++) {
```

Comment:
La experiencia y el dolor y sufrimiento me han enseñado a evitar los `for` en javascript y usar sus equivalentes más funcionales... Queda mucho más legible el código.

Te propongo refactorizar el método usando `Array.filter()` y `Array.forEach()`